### PR TITLE
Update gui_de.properties

### PR DESCRIPTION
**Pull request by Benni76**
german language grammar error fixed

### CODE REVIEW

File: src/main/resources/resources/logisim/strings/gui/gui_de.properties [hunk 1]
```diff
@@ -578,8 +578,8 @@ simStrokeColTitle = Farbe für Leitungen/Buse:
 simTitle = Simulation
 simTrueCharTitle = Zeichen für True Wert:
 simTrueColTitle = Farbe für True:
-simUnknownCharTitle = Zeichen für unbekante Werte:
-simUnknownColTitle = Farbe für unbekante:
+simUnknownCharTitle = Zeichen für unbekannte Werte:
```
Comment (maehne) [inline src/main/resources/resources/logisim/strings/gui/gui_de.properties:581]: "Zeichen für unbekannten Wert:"

File: src/main/resources/resources/logisim/strings/gui/gui_de.properties [hunk 2]
```diff
@@ -578,8 +578,8 @@ simStrokeColTitle = Farbe für Leitungen/Buse:
...
+simUnknownColTitle = Farbe für unbekannte:
```
Comment (maehne) [inline src/main/resources/resources/logisim/strings/gui/gui_de.properties:582]: "Farbe für unbekannten Wert:"

**Review by maehne — CHANGES_REQUESTED** [review]
Thanks for noticing the typo and proposing fixes!

File: src/main/resources/resources/logisim/strings/gui/gui_de.properties (hunk 1 above)
Comment (Benni76) [reply src/main/resources/resources/logisim/strings/gui/gui_de.properties]: Fixed

File: src/main/resources/resources/logisim/strings/gui/gui_de.properties (hunk 2 above)
Comment (Benni76) [reply src/main/resources/resources/logisim/strings/gui/gui_de.properties]: Fixed

**Comment by maehne** [comment]
We need to check for potential conflicts to PR #1870, i.e., merge this one after it.

File: src/main/resources/resources/logisim/strings/gui/gui_de.properties [hunk 3]
```diff
@@ -564,11 +564,11 @@ preferencesFrameTitle = Voreinstellungen
 #
 simBusColTitle = Bus Farbe:
 simColorBlindColors = Farbenblinden Farben
-simDefaultColors = Logisim-evolution spandart Farben
+simDefaultColors = Logisim-evolution standard Farben
```
Comment (maehne) [inline src/main/resources/resources/logisim/strings/gui/gui_de.properties:567]: "standard Farben" -> "Standardfarben"

**Review by maehne — APPROVED** [review]
Looks mostly good except for one translation.

**Pull request merged**